recipes/playwright/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "playwright" %}
+{% set version = "1.11.0a2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kumaraditya303/playwright-python/archive/refs/tags/v1.11.0.a3.tar.gz
+  sha256: 01623da68cab02468d53250342a306199313d737c786887d415b2b067f52ff34
+
+build:
+  script_env:
+    - SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: true  # [py<37]
+  binary_relocation: False
+  missing_dso_whitelist: "*"
+  entry_points:
+    - playwright = playwright.__main__:main
+    
+requirements:
+  build:
+    - wheel
+    - curl
+    - setuptools_scm
+  host:
+    - python
+    - pip
+    - greenlet
+    - pyee
+    - curl
+    - setuptools_scm
+    - websockets
+    - typing_extensions
+  run:
+    - python
+    - greenlet
+    - pyee
+    - websockets
+    - typing_extensions
+
+test:
+  imports:
+    - playwright
+    - playwright.sync_api
+    - playwright.async_api
+  commands:
+    - playwright --help
+
+about:
+  home: https://github.com/microsoft/playwright-python
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.txt
```

```suggestion
  license_file: LICENSE
```

As the source code comes from a GitHub archive, the _official_ license file is contained within it. So you should remove the license file in this recipe a link to the `LICENSE` file from the GitHub archive.